lib/brewdler/brew_dumper.rb

```diff
@@ -0,0 +1,42 @@
+require "json"
+require "version"
```

Comment:
The `opt` symlink should always be present whereas `linkedkeg` may not be. If it's useful we could consider adding more to the JSON output, here.